### PR TITLE
gh-180: add badges for pre-commit, python versions, discussion, license

### PR DESCRIPTION
**Pull request by Saransh-cpp**
Refs: #180

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
 # **GLASS**: Generator for Large Scale Structure
 
 [![Test](https://github.com/glass-dev/glass/actions/workflows/test.yml/badge.svg)](https://github.com/glass-dev/glass/actions/workflows/test.yml)
+[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/glass-dev/glass/main.svg)](https://results.pre-commit.ci/latest/github/glass-dev/glass/main)
 [![Documentation](https://readthedocs.org/projects/glass/badge/?version=latest)](https://glass.readthedocs.io/latest/)
 [![Coverage Status](https://coveralls.io/repos/github/glass-dev/glass/badge.svg?branch=main)](https://coveralls.io/github/glass-dev/glass?branch=main)
 [![PyPI](https://img.shields.io/pypi/v/glass)](https://pypi.org/project/glass)
+[![PyPI platforms][https://img.shields.io/pypi/pyversions/glass]][https://pypi.org/project/glass/]
```
Comment (ntessore) [inline README.md:8]:
```suggestion
[![PyPI platforms](https://img.shields.io/pypi/pyversions/glass)](https://pypi.org/project/glass/)
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 [![arXiv](https://img.shields.io/badge/arXiv-2302.01942-red)](https://arxiv.org/abs/2302.01942)
 [![adsabs](https://img.shields.io/badge/ads-2023OJAp....6E..11T-blueviolet)](https://ui.adsabs.harvard.edu/abs/2023OJAp....6E..11T)
 [![doi](https://img.shields.io/badge/doi-10.21105/astro.2302.01942-blue)](https://dx.doi.org/10.21105/astro.2302.01942)
+[![LICENSE][https://img.shields.io/badge/License-MIT-blue.svg]][https://opensource.org/licenses/MIT]
```
Comment (ntessore) [inline README.md:12]:
```suggestion
[![LICENSE](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
```

File: README.md [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
+[![GitHub Discussion][https://img.shields.io/static/v1?label=Discussions&message=Ask&color=blue&logo=github]][https://github.com/orgs/glass-dev/discussions]
```
Comment (ntessore) [inline README.md:13]:
```suggestion
[![GitHub Discussion](https://img.shields.io/static/v1?label=Discussions&message=Ask&color=blue&logo=github)](https://github.com/orgs/glass-dev/discussions)
```

**Review by ntessore — CHANGES_REQUESTED** [review]
Some badges aren't rendering

**Comment by Saransh-cpp** [comment]
Oops, my bad! Thanks for pointing out.

**Review by ntessore — APPROVED** [review]

**Pull request merged**